neurom/apps/morph_stats.py

```diff
@@ -152,12 +139,13 @@ def extract_stats(neurons, config):
         neurons: a neuron, population, neurite tree or list of neuron paths/str
         config (dict): configuration dict. The keys are:
             - neurite_type: a list of neurite types for which features are extracted
-              If not provided, all neurite_type will be used
+              If not provided, all neurite_type will be used.
             - neurite: a dictionary {{neurite_feature: mode}} where:
-                - neurite_feature is a string from NEURITEFEATURES
+                - neurite_feature is a string from NEURITEFEATURES or NEURONFEATURES
```

Comment:
not NEURONFEATURES

Reply:
both types of features works in both neurite or neuron entries. only the neuron entry will not aply to all neurite_types